easybuild/easyconfigs/i/icc/icc-2015.3.187-GCCcore-4.9.3-binutils-2.25.eb

```diff
@@ -0,0 +1,27 @@
+name = 'icc'
+version = '2015.3.187'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_ccompxe_%(version)s.tgz']
+
+gcc = 'GCCcore'
+binutilsver = '2.25'
+gccver = '4.9.3-binutils-%s' % binutilsver
```

Comment:
I would keep using `GCC` in the versionsuffix rather than `GCCcore`?

`GCCcore` should be an internal thing as much as possible (maybe even a hidden module by default?)